soc/arm/st_stm32/stm32l4/Kconfig.defconfig.stm32l471xx

```diff
@@ -0,0 +1,37 @@
+# Kconfig - ST Microelectronics STM32L476RG MCU
+#
+# Copyright (c) 2018 Allterco Robotics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32L471XG
```

Comment:
Can you rename to SOC_STM32L471X**X**?
Package information (last letter of the reference name) is not needed anymore since we extract package sizes from dts